Alert

chanchal-panpaliya opened the pull request:
project live - https://solutionground.netlify.app/

chanchal-panpaliya committed:
alert component commit

alert component with responsive for component library

chanchal-panpaliya committed:
alert component with desktop and mobile screens 

alert component with desktop and mobile screens  for the component library .

chanchal-panpaliya committed:
netify linked aaded

netify linked added

Mohasin-Haque reviewed (commented):
Your code is good but you need to work on sanity. Remove the whitespaces and commented codes.

chanchal-panpaliya committed:
change px -> rem and remove unwanted space

change px -> rem and remove unwanted space.

chanchal-panpaliya committed:
removed space , color ,layout changed , added css iframe

removed space , color ,layout changed , added css iframe

rishbitsnbytes commented on Alert/Alert.css:
```diff
@@ -0,0 +1,134 @@
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    z-index: 999;
```
Z-Index, not a problem in this 999 but better to use sequential like 5-6 etc. Because as project gets bigger these random big indices confuses a lot. And for other team members who might be working on with you on some project too it's not good. I read it in a blog. Give it a thought.

chanchal-panpaliya replied:
ok. i will change this. Thank you so much for pointing this.

chanchal-panpaliya replied:
removed z-index

rishbitsnbytes commented on Alert/Alert.css:
```diff
@@ -0,0 +1,134 @@
+#alert_example{
+    display: none;
+    position: absolute;
+    z-index: 1;
```
Two Z-index in same class ? Might be a mistake. Pls recheck

rishbitsnbytes commented on Alert/Alert.css:
```diff
@@ -0,0 +1,134 @@
+
+/*---alert example---*/
+
+#show_alert{
```
As Vikram also mentioned once. Consistency not only in UI but in everything even in our code is a must. Use either classes or IDs.

chanchal-panpaliya replied:
It's just to change the priority of the "live example" alert component. I will update my code.

chanchal-panpaliya replied:
changed ID -> Class

rishbitsnbytes commented on Alert/Alert.css:
```diff
@@ -0,0 +1,134 @@
+
+
+.alert-outline-success{
+    border : 1px solid var(--green);
```
I think, the purpose of using variables is we can change colors as per our needs. If we name them like this then no point of using variable. So, better do naming which doesn't block it's future possibilities of changes   like here you can create and use var(--alert-success-border)

chanchal-panpaliya replied:
done.

xii-vi reviewed (approved):
Code is clean and properly structured. You can merge

rishbitsnbytes commented on Alert/Alert.css:
```diff
@@ -0,0 +1,134 @@
+/*--------fill---alert*/
+.alert-fill-box-success{
+  color:var(--green);
+  background-color: var(--background-color-green);
```
Same as mentioned above, better variable names and ad short as possible, like for this instance you can use var(--alert-success-bg)

chanchal-panpaliya replied:
ok

rishbitsnbytes reviewed (commented):
Added few comments, Rest looks good to me.

chanchal-panpaliya committed:
removed id , z-index , add alert utility class (separate)

removed id , z-index , added alert utility class - bordercolor , bgcolor , color (update css code).JS code update.
